tests/data/charm.yaml

```
@@ -0,0 +1,25 @@
+description: Overlay for attaching current charm
+series: {{ series }}
+applications:
+  aws-integrator:
+    charm: {{ charm }}
+    channel: null
+    resources:
+      aws-cli: 0
+    num_units: 1
+    trust: True
```

```suggestion
    trust: true
```

It'll work with `T`, but lc is consistent with our other test data and overlays.